dotnet/dotnet-guestbook/.readmes/intellij/README.md

```diff
@@ -119,7 +125,7 @@ You can see how the Guestbook modules are defined by checking out the frontend's
 For more info on how to use Skaffold modules, see the [Skaffold documentation](https://skaffold.dev/docs/design/config/#multiple-configuration-support).
 
 ---
-### Run individual services with Skaffold modules
+<h3 id="run-individual-services-with-skaffold-modules"> Run individual services with Skaffold modules </h3>
```

Comment:
This is the same Id as above

Reply:
oh yep, you totally caught problems with the ids! Great catch @averikitsch

Reply:
I'll make a separate PR to fix this - the entire "Run individual services with Skaffold modules" section is included twice, and I'll remove the duplicate from each readme